skyportal/models/obj.py

```diff
@@ -406,8 +406,8 @@ def add_linked_thumbnails(self, session=DBSession):
         """Determine the URLs of the SDSS and DESI DR8 thumbnails of the object,
         insert them into the Thumbnails table, and link them to the object."""
         sdss_thumb = Thumbnail(obj=self, public_url=self.sdss_url, type='sdss')
-        dr8_thumb = Thumbnail(obj=self, public_url=self.desi_dr8_url, type='dr8')
-        session.add_all([sdss_thumb, dr8_thumb])
+        dr9_thumb = Thumbnail(obj=self, public_url=self.desi_dr9_url, type='dr9')
```

Comment:
Would it be smarter to name the types based on the underlying survey, rather than data release? Wouldn't DESI be a better name than dr9?

Reply:
I realize this is how it was until now, but this is actually a good example why naming things for DR's is not the best idea...

Reply:
great idea. we will rename it.